public/v4/apps/windmill.yml

```diff
@@ -19,28 +19,25 @@ services:
         environment:
             DATABASE_URL: postgres://postgres:$$cap_postgres_pass@srv-captain--$$cap_appname-postgres/windmill?sslmode=disable
             BASE_URL: $$cap_wm_base_url
-            RUST_LOG: 'info'
-            NUM_WORKERS: '0'
-            DISABLE_SERVER: 'false'
-            METRICS_ADDR: 'false'
+            RUST_LOG: info
+            NUM_WORKERS: 0
         depends_on:
             - $$cap_appname-postgres
         caproverExtra:
-            containerHttpPort: '8000'
+            containerHttpPort: 8000
 
     $$cap_appname-worker:
         image: ghcr.io/windmill-labs/windmill:$$cap_app_version
+        volumes:
+            - $$cap_appname-worker-dependency-cache:/tmp/windmill/cache
```

Comment:
I'm not familiar with Windmill, but is making the cache persistent really necessary? Usually caches are supposed to be transient as the name suggests....

Reply:
I'm following the original `docker-file`. In the previous one-click version, before having the `docker socket`, I did not pay attention to this. But now with the docker socket, we do have the need of persistence and the caching is supposed to speed up all the packages installs, so I added it.